mappings/net/minecraft/client/gui/widget/AbstractPressableButtonWidget.mapping

```diff
@@ -1,2 +1,3 @@
 CLASS net/minecraft/class_4264 net/minecraft/client/gui/widget/AbstractPressableButtonWidget
```

Comment:
The distinction between being clickable and being pressable seems a bit unclear, might be worth renaming this?